praw/__init__.py

```diff
@@ -2300,6 +2300,50 @@ def send_message(self, recipient, subject, message, from_sr=None,
         return response
 
 
+class ReportMixin(AuthenticatedReddit):
+
+    """Adds methods requiring the 'report' scope (or login).
+
+    You should **not** directly instantiate instances of this class. Use
+    :class:`.Reddit` instead.
+
+    """
+
+    @decorators.restrict_access(scope='report')
+    def hide(self, thing_id, unhide=False):
+        """Hide up to 50 objects in the context of the logged in user.
+
+        :param thing_id: A single fullname or list of fullnames,
+            representing objects which will be hidden.
+        :param unhide: If True, unhide the object(s) instead.
+
+        :returns: The json response from the server.
+
+        """
+        if not isinstance(thing_id, six.string_types):
+            thing_id = ','.join(thing_id)
+        method = 'unhide' if unhide else 'hide'
+        url = self.config[method]
+        data = {'id': thing_id,
+                'executed': method}
+        response = self.request_json(url, data=data)
+        urls = [urljoin(self.user._url, 'hidden')]
```

Comment:
Which variable is unecessary? You made a note about the `method` variable above, but scratched it out. Is this note referring to the same variable?